lib/lndclient/LndClient.ts

```diff
@@ -195,12 +212,16 @@ class LndClient extends BaseClient {
       })
       .on('end', () => {
         this.logger.info('invoice ended');
+        this.setStatus(ClientStatus.DISABLED);
+        this.connect(this);
```

Comment:
When `connect` is called here loading the certificate and macaroon again isn't sensible. Initializing `lndClient` should be moved back to the constructor.